README.md

```diff
@@ -63,19 +71,23 @@ The local API access needs to be activated once in the settings of the EZ1. Plea
 </ul>
 
 ---
+
 ## Installation
+
 - To use the APsystemsEZ1 library, you need to have Python >=3.8 installed on your system.
-- See the following guide to install the latest Python release: https://www.python.org/downloads <br><br>
+- See the following guide to install the latest Python release: <https://www.python.org/downloads> <br><br>
```

Comment:
Why did you put the Links into the `<` and `>`?

Reply:
This ist following `MD034` rule for markdown linting: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md034---bare-url-used